civihr_employee_portal/features/civihr_default_permissions/civihr_default_permissions.features.user_permission.inc

```diff
@@ -36,9 +36,9 @@ function civihr_default_permissions_user_default_permissions() {
   $permissions['access CiviCRM'] = array(
     'name' => 'access CiviCRM',
     'roles' => array(
+      'HR Admin' => 'HR Admin',
+      'Regional HR Admin' => 'Regional HR Admin',
```

Comment:
I thought this array was something like "machine_name" => "name". Why would someone need an array where the keys and the values are the same??????

Reply:
Thought same too, doesn't seem to make much sense, Was when I re-exported the feature that I saw it that way too